## Detecting genetic structural variation in the human genome in bulk experiments and single cells

Structural variants (SVs) are sequence variants of at least 50 base pairs in size including deletions, duplications, insertions, inversions and translocations. Genomic structural variants have been implicated in phenotypic diversity and human disease. SVs can be shared by all the cells in human body (germline), or arise in the soma of an individual (somatic) which makes human body as complex mosaics of genetically distinct cells. In this course, we will talk about the concepts of current genomics technologies in bulk (whole genome sequencing) and single-cell level (Strand-seq scTRIP) to detect SVs genome-wide.  We will practice how to analyze those bulk and single-cell genomics data using bioinformatics softwares such as Delly, Snakemake pipeline, BreakpointR, and Mosaicatcher. Eventually we will discuss what kind of biological questions we can address with the help of those technologies and bioinformatics tools.

<Suggested reading - biological>

Forsberg et al, Nature Reviews Genetics 2017: “Mosaicism in health and disease — clones picking up speed” https://www.nature.com/articles/nrg.2016.145
Chaisson et al, Nature Communications, 2019: “Multi-platform discovery of haplotype-resolved structural variation in human genomes” https://www.nature.com/articles/s41467-018-08148-z

<Suggested reading - computational>
Rausch et al, Bioinformatics, 2012: "DELLY: structural variant discovery by integrated paired-end and split-read analysis"
https://academic.oup.com/bioinformatics/article/28/18/i333/245403
Porubsky et al, Bioinformatics, 2019: "breakpointR: an R/Bioconductor package to localize strand state changes in Strand-seq data" https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz681/5556813
